int32 action
float32 x
float32 y
---
int32 status
float32 x_current
float32 y_current
